open-hackathon-server/src/hackathon/storage/azure_storage.py

```diff
@@ -62,7 +64,11 @@ def save(self, context):
         if context.get('content'):
             file_content = context.content
             if isinstance(file_content, file) or isinstance(file_content, FileStorage):
+                # encrypt certification file before upload to azure.
+                file_content = self.__encrypt_content(context)
```

Comment:
The `storage` module handles content saving and retrieving but not content formatting/converting. It saves content from caller but should have no idea what the content is or that it represents.  Otherwise you have to encrypt/decrypt contents in every implementations of `Storage`. For example, in your implementation, the `LocalStorage` may not work. And developer must fully understand the cert mechanism even if they just want to add a type of Storage, for example AWS S3.